metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsExperiment.groovy

```diff
@@ -2,14 +2,20 @@ package life.qbic.metastats.datamodel
 
 class MetaStatsExperiment {
 
-    final String type
-    List<String> samples
+    final String experimentType
+    List<String> relatedSampleCodes
     Map<String, String> properties
```

Comment:
This conflicts with default groovy methods https://groovy-lang.org/objectorientation.html#properties

Reply:
You are right, resolved it with 084924b